lms/djangoapps/teams/toggles.py

```diff
@@ -0,0 +1,43 @@
+"""
+Togglable settings for Teams behavior
+"""
+from edx_toggles.toggles import SettingDictToggle
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
+
+
+# Course Waffle inherited from edx/edx-ora2
+WAFFLE_NAMESPACE = "openresponseassessment"
+TEAM_SUBMISSIONS_FLAG = "team_submissions"
+
+# .. toggle_name: FEATURES['ENABLE_ORA_TEAM_SUBMISSIONS']
+# .. toggle_implementation: SettingDictToggle
+# .. toggle_default: False
+# .. toggle_description: Set to True to enable team-based ORA submissions.
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2020-03-03
+# .. toggle_target_removal_date: None
+# .. toggle_tickets: https://openedx.atlassian.net/browse/EDUCATOR-4951
+# .. toggle_warnings: This temporary feature toggle does not have a target removal date. This can be overridden by a
+#      course waffle flags or a waffle switch with identical name.
+# TODO: this should be moved to edx/edx-ora2
+TEAM_SUBMISSIONS_FEATURE = SettingDictToggle(
+    "FEATURES", "ENABLE_ORA_TEAM_SUBMISSIONS", default=False, module_name=__name__
+)
+
+
+def are_team_submissions_enabled(course_key):
+    """
+    Checks to see if the CourseWaffleFlag or Django setting for team submissions is enabled
+    """
+    if CourseWaffleFlag(WAFFLE_NAMESPACE, TEAM_SUBMISSIONS_FLAG, __name__).is_enabled(
+        course_key
+    ):
+        return True
+
+    # TODO: this behaviour differs from edx-ora2, where the WaffleSwitch overrides the setting.
+    # https://github.com/edx/edx-ora2/blob/ac502d8301cb987c9885aaefbaeddaf456c13fb9/openassessment/xblock/config_mixin.py#L96
```

Comment:
@iloveagent57: FYI: This seems like a bug, but it is unclear if anyone is using switches for this (or any of the other flags) in ORA. For edx.org Prod, it looks like we are only using the setting. I'm hoping we can eventually:
* Move ORA setting definitions to edx/edx-ora2.
* Have a single setting type for each of these named settings.  Use a setting if we want on/off.  Kill the switch.  Use a CourseWaffleFlag instead if we want to be able to turn on/off for certain courses or users.
* If any of these could be killed altogether, that would be best. :)
Mostly, this is just an FYI though.  I'm not sure whether your team even owns ORA.  :)

Reply:
@edx/masters-devs-gta is the ORA owner of record :). Adding for notification.